### PR TITLE
Bug 1729524 Add adm_export weekly DAG

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1729524

Uses new docker image from https://github.com/mozilla/docker-etl/pull/28

Depends on https://github.com/mozilla/bigquery-etl/pull/2373

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -52,6 +52,7 @@ services:
       - AIRFLOW__KUBERNETES__IN_CLUSTER=False
       - URL=http://localhost:8000
       - WEBSERVER_USE_RBAC=False
+      - 'AIRFLOW_CONN_ADM_SFTP=ftp://myname:mypassword@myhost.com:8000?known_hosts=myhost.com+AAAABBBBB'
```
Comment (jklukas) [inline docker-compose.yml:55]: I'm adding dummy connection details for the "adm_sftp" connection as env var here to allow the dag parsing test to run successfully. This appears to be working, and may be quicker to load than using the CLI to add the connection in `bin/run`. AFAICT, there's overhead to every airflow CLI call that's dominating the startup time.

File: dags/adm_export.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+) as dag:
+
+    conn = BaseHook.get_connection('adm_sftp')
+
+    adm_weekly_aggregates_to_sftp = GKEPodOperator(
+        task_id="adm_weekly_aggregates_to_sftp",
+        name="adm_weekly_aggregates_to_sftp",
+        # See https://github.com/mozilla/docker-etl/pull/28
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/bq2sftp_docker_etl:latest",
+        project_id='moz-fx-data-shared-prod',
```
Comment (haroldwoo) [inline dags/adm_export.py:43]: I think the project id and the gcp conn id need to correspond, else the operator will think the workloads-prod-v1 gke cluster exists in shared-prod.

File: docker-compose.yml (hunk 1 above)
Comment (haroldwoo) [reply docker-compose.yml:55]: This was an option i considered as well. It becomes a little more complex with gcp/aws type connections, and we can use a .env file to store all of them as well. 

https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#storing-a-connection-in-environment-variables

Why is the whole k:v in single quotes? Shouldn't it be k='v'?

**Review by haroldwoo — APPROVED** [review]
lgtm. had some comments

File: docker-compose.yml (hunk 1 above)
Comment (jklukas) [reply docker-compose.yml:55]: From the yaml document's perspective, this is a list of strings. It's docker-compose that is expecting to parse these items as `k=v`. If we wrote `k='v'` here, then the value would literally contain the quotes.

That said, I don't think there are any characters here that are problematic, so we don't need any quotes at all.

File: dags/adm_export.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+) as dag:
+
+    conn = BaseHook.get_connection('adm_sftp')
+
+    adm_weekly_aggregates_to_sftp = GKEPodOperator(
+        task_id="adm_weekly_aggregates_to_sftp",
+        name="adm_weekly_aggregates_to_sftp",
+        # See https://github.com/mozilla/docker-etl/pull/28
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/bq2sftp_docker_etl:latest",
+        project_id=GoogleCloudBaseHook(gcp_conn_id=gcp_gke_conn_id).project_id,
```
Comment (haroldwoo) [inline dags/adm_export.py:44]: i dont see gcp_gke_conn_id defined?

Comment (haroldwoo) [reply dags/adm_export.py:44]: if this needs to be revised, can you just hardcode it to 'moz-fx-data-airflow-gke-prod', since 2.0 will need it this way

Comment (jklukas) [reply dags/adm_export.py:44]: You're right. I copy/pasted this. I had set auto-merge and I was expecting this to not merge if the test-environment task failed. Apologies for the churn. I'm working on a follow-up immediately.

**Pull request merged**